content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -7,73 +7,86 @@ reviewers:
 - liggitt
 content_type: concept
 weight: 25
-min-kubernetes-server-version: 1.16
 ---
 
 <!-- overview -->
 
 {{< feature-state for_k8s_version="v1.22" state="stable" >}}
 
-## Introduction
+Kubernetes supports multiple appliers collaborating to manage the `.spec` and / or
+`.metadata` of a single [object](/docs/concepts/overview/working-with-objects/).
 
-Server Side Apply helps users and controllers manage their resources through
-declarative configurations. Clients can create and modify their
+Server-Side Apply provides a mechanism for your cluster's control plane to track
+changes to an object's fields. At the level of a specific resource, Server-Side
+Apply records and tracks optional information about control over the fields
+of that object.
+
+<!-- body -->
+
+## Motivation
+
+Server-Side Apply helps users and controllers manage their resources through
+declarative configuration. Clients can create and modify their
 [objects](/docs/concepts/overview/working-with-objects/kubernetes-objects/)
 declaratively by sending their fully specified intent.
 
 A fully specified intent is a partial object that only includes the fields and
 values for which the user has an opinion. That intent either creates a new
 object or is [combined](#merge-strategy), by the server, with the existing object.
 
-The system supports multiple appliers collaborating on a single object.
-
-Changes to an object's fields are tracked through a "[field management](#field-management)"
-mechanism. When a field's value changes, ownership moves from its current
-manager to the manager making the change. When trying to apply an object,
-fields that have a different value and are owned by another manager will
-result in a [conflict](#conflicts). This is done in order to signal that the
-operation might undo another collaborator's changes. Conflicts can be forced,
-in which case the value will be overridden, and the ownership will be
-transferred.
+## Field management
 
-If you remove a field from a configuration and apply the configuration, server
-side apply checks if there are any other field managers that also own the
-field.  If the field is not owned by any other field managers, it is either
-deleted from the live object or reset to its default value, if it has one. The
-same rule applies to associative list or map items.
+If you have Server-Side Apply enabled, the Kubernetes control plane tracks
```

Comment:
That's true by default, possibly the option of disabling will be removed, should we make it clearer that this is the default?

Reply:
I can add that it's enabled by default.